rllib/agents/mbmpo/utils.py

```diff
@@ -66,3 +77,50 @@ def discount_cumsum(x, discount):
         """
     return scipy.signal.lfilter(
         [1], [1, float(-discount)], x[::-1], axis=0)[::-1]
+
+
+class MBMPOExploration(Exploration):
+    """An exploration that simply samples from a distribution.
```

Comment:
We should simply add the n-initial-random timesteps option to the existing StochasticSampling Exploration class (similar to GaussianNoise and OrnsteinUhlenbeck). Then all algos that currently use StochasticSampling could benefit from this as well. Or is there another reason, why MBMPO Exploration is special?

Reply:
It is because only the local worker (not remote workers) needs the random sampling for action space!

Reply:
Got it. Makes sense. Thanks for clarifying.